editors/sc-ide/CMakeLists.txt

```diff
@@ -193,11 +193,14 @@ set(ide_webengine_moc_hdrs
     widgets/util/WebSocketClientWrapper.hpp
     widgets/util/IDEWebChannelWrapper.hpp
     ${CMAKE_SOURCE_DIR}/QtCollider/widgets/web_page.hpp
+    ${CMAKE_SOURCE_DIR}/QtCollider/widgets/QcWebView.h
+    ${CMAKE_SOURCE_DIR}/QtCollider/QcCallback.hpp
 )
 set(ide_webengine_src
     widgets/help_browser.cpp
     widgets/util/WebSocketTransport.cpp
     ${CMAKE_SOURCE_DIR}/QtCollider/widgets/web_page.cpp
+    ${CMAKE_SOURCE_DIR}/QtCollider/widgets/QcWebView.cpp
```

Comment:
not something you can address in this PR but i think we need 1 or 2 new static libraries in the project for the common Qt code (the stuff being added here) and common non-Qt code (the stuff in common/)